README.md

```diff
@@ -34,3 +34,42 @@ Shuttle Tracker needs configuration to run properly. The preferred method during
 ### Environment variables
 
 Most keys can be overridden with environment variables. The variables names usually take the format `SECTION_KEY`. For example, overriding database's Mongo URL could be done with a variable named `POSTGRES_URL`.
+
+
+## Setting up (Windows)
+
+1. [Download Go](https://golang.org/dl/). Shuttle Tracker targets Go version 1.7 and newer, but we recommend using the latest stable release of Go.
+![Step 1](https://user-images.githubusercontent.com/6248819/47017483-5e891180-d120-11e8-9157-6c5823ecb13a.png)
+2. Ensure your `$GOPATH` is set correctly, and `$GOPATH/bin` is in your `$PATH`, this should be done for you already.
+![Step 2](https://user-images.githubusercontent.com/6248819/47017509-71034b00-d120-11e8-82ee-01b17afb7ec0.png)
+![Step 2b](https://user-images.githubusercontent.com/6248819/47017534-811b2a80-d120-11e8-9a8a-a625b8a74e02.png)
+3. Open a command prompt by pressing windows + r, then type `cmd` and hit ok, or search for command prompt
+![Step 3](https://user-images.githubusercontent.com/6248819/47017557-9001dd00-d120-11e8-8258-651745338d78.png)
+4. Run `go get github.com/wtg/shuttletracker`
+5. Install `govendor` by running `go get -u github.com/kardianos/govendor`
+6. Switch to the Shuttle Tracker directory (`$GOPATH/src/github.com/wtg/shuttletracker`)
+7. Run `govendor sync`
+![Steps 4-6](https://user-images.githubusercontent.com/6248819/47017579-9db76280-d120-11e8-8de5-ab5cbe11e072.png)
+8. Download and run the installer PostgreSQL [from here](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads). Select the latest version. When prompted to set a password, make it something simple, as you will be using this later, for example `shuttle`.  Use default options for everything else. Write down your password.
+![Step 8](https://user-images.githubusercontent.com/6248819/47017593-ac057e80-d120-11e8-8637-18307ebeaf7e.png)
+![Step 8b](https://user-images.githubusercontent.com/6248819/47017613-b6c01380-d120-11e8-85ee-35442f6ea737.png)
+![Step 8c](https://user-images.githubusercontent.com/6248819/47017633-c2abd580-d120-11e8-95ab-50088fb38c40.png)
+9. When complete, open pgAdmin from your search bar. In the Object Browser, open Servers, then open your PostgreSQL server. You will need to enter your password from step 8. Once this is done, right click on Databases and select New Database. Name it `shuttletracker` and hit Ok.
```

Comment:
May want to add that if a user doesn't see pgAdmin in their search after postgres is installed that they should restart their computer.